support-frontend/assets/components/paymentMethodSelector/PaymentMethodSelectorContainer.tsx

```diff
@@ -70,12 +74,19 @@ function PaymentMethodSelectorContainer({
 	const { switches } = useContributionsSelector(
 		(state) => state.common.settings,
 	);
+	const participations: Participations = initAbTests(
+		detectCountry(),
+		countryGroupId,
+		getSettings(),
+	);
```

Comment:
We could use the contributions selector here to get the participations from state

```
	const { abParticipations } = useContributionsSelector(
		(state) => state.common,
	);